### ISSUE TITLE
尊敬的作者，请问simA和simB是没有后续的作用吗？

### ISSUE THREAD

**Issue by kapbl** [QinYang79/RDE#15]
lossA, lossB, **simsA,simsB** = torch.zeros(data_size), torch.zeros(data_size), torch.zeros(data_size),torch.zeros(data_size)
    for i, batch in enumerate(data_loader):
        batch = {k: v.to(device) for k, v in batch.items()}
        index = batch['index']
        with torch.no_grad(): 
            la, lb, sa, sb = model.compute_per_loss(batch)
            for b in range(la.size(0)):
                lossA[index[b]]= la[b]
                lossB[index[b]]= lb[b]
                **simsA[index[b]]= sa[b]
                simsB[index[b]]= sb[b]**
            if i % 100 == 0:
                logger.info(f'compute loss batch {i}')

**Comment by QinYang79** [comment]
> lossA, lossB, **simsA,simsB** = torch.zeros(data_size), torch.zeros(data_size), torch.zeros(data_size),torch.zeros(data_size) for i, batch in enumerate(data_loader): batch = {k: v.to(device) for k, v in batch.items()} index = batch['index'] with torch.no_grad(): la, lb, sa, sb = model.compute_per_loss(batch) for b in range(la.size(0)): lossA[index[b]]= la[b] lossB[index[b]]= lb[b] **simsA[index[b]]= sa[b] simsB[index[b]]= sb[b]** if i % 100 == 0: logger.info(f'compute loss batch {i}')

Thanks for your attention to our work! It is only used to record similarities to understand the distribution.
